Makefile.in

```diff
@@ -0,0 +1,134 @@
+#
+# Makefile rules for the datastructures package
+#
+# This requires GNU make (just as the main GAP does).
+#
+# For simple packages one should only need to modify the PKG_SO_NAME
+# and the SOURCES variable.
+#
+PKG_SO_NAME = datastructures.so
```

Comment:
I think on Windows, this needs to be `datastructures.dll`. At least it was in the old build system. That would also suggest removing the suffix here entirely, and instead putting it into the "common" build rules below. And then perhaps rename `PKG_SO_NAME` -- e.g. to `PACKAGE_NAME`, or `MODULE_NAME` or `KERNEL_EXTENSION_NAME` or so?